app/main.py

```diff
@@ -1,2 +1,70 @@
+from collections.abc import Hashable
+from typing import Any
+
+
+class Node:
```

Comment:
This class better be defined inside of a Dictionary since it is a part of its implementation and will not be used outside of it